src/target-framework.props

```diff
@@ -1,12 +1,12 @@
 <Project>
 
-  <PropertyGroup Condition="$(NETCoreSdkVersion.StartsWith('3.0'))">
+  <PropertyGroup Condition="!$(NETCoreSdkVersion.StartsWith('2.1'))">
     <_UseNetCoreApp3>true</_UseNetCoreApp3>
   </PropertyGroup>
 
   <PropertyGroup>
     <TargetFramework Condition="$(_UseNetCoreApp3) == true">netcoreapp3.0</TargetFramework>
-	<TargetFramework Condition="$(_UseNetCoreApp3) != true">netcoreapp2.1</TargetFramework>   
+    <TargetFramework Condition="$(_UseNetCoreApp3) != true">netcoreapp2.1</TargetFramework>   
```

Comment:
Is this white space change intentional?

Reply:
Sorry, I read the change backwards. Please ignore the question.

Reply:
Yeah, I replaced a tab with spaces